packages/@uppy/drop-target/types/index.d.ts

```diff
@@ -2,6 +2,9 @@ import type { PluginOptions, BasePlugin } from '@uppy/core'
 
 interface DropTargetOptions extends PluginOptions {
     target: string | Element
+    activeClass?: string
+    setMeta: (file: File, event: DragEvent) => Object
```

Comment:
We should use a more descriptive type than `Object`, and mark the option as optinal.
```suggestion
    setMeta?: (file: File, event: DragEvent) => Record<string, unknown>
```